src/Pim/Component/Catalog/Normalizer/Indexing/Product/MetricNormalizer.php

```diff
@@ -0,0 +1,43 @@
+<?php
+
+namespace Pim\Component\Catalog\Normalizer\Indexing\Product;
+
+use Pim\Component\Catalog\AttributeTypes;
+use Pim\Component\Catalog\Model\ProductValueInterface;
+use Pim\Component\Catalog\ProductValue\MetricProductValue;
+use Symfony\Component\Serializer\Normalizer\NormalizerInterface;
+
+/**
+ * Normalizer for a text (simple text) product value
+ *
+ * @author    Samir Boulil <samir.boulil@akeneo.com>
+ * @copyright 2017 Akeneo SAS (http://www.akeneo.com)
+ * @license   http://opensource.org/licenses/osl-3.0.php Open Software License (OSL 3.0)
+ */
+class MetricNormalizer extends AbstractProductValueNormalizer implements NormalizerInterface
+{
+    /**
+     * {@inheritdoc}
+     */
+    public function supportsNormalization($data, $format = null)
+    {
+        return $data instanceof MetricProductValue &&
+            AttributeTypes::BACKEND_TYPE_METRIC === $data->getAttribute()->getBackendType() &&
```

Comment:
I'd say yes, because two different attributes can be mapped on the same back-end type